lib/internal/domexception.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const { internalBinding } = require('internal/bootstrap/loaders');
+const { registerDOMException } = internalBinding('messaging');
+const { ERR_INVALID_THIS } = require('internal/errors').codes;
+
+const internalsMap = new WeakMap();
+
+const nameToCodeMap = new Map();
+
+class DOMException extends Error {
```

Comment:
Should there be anything added to the documentation about this new error type?

Reply:
Not sure, but I'm inclined to say no, as it's pretty different from rest of `errors.md`, and `DOMException` is not exposed to userland by default.